packages/primero/test/index.js

```diff
@@ -88,3 +103,97 @@ describe.skip('The getCases() function', () => {
     });
   });
 });
+
+describe('http.get', () => {
```

Comment:
Can we move all the http helpers into a file called `http.test.js`? It's way tidier